### DISCUSSION TITLE
How can I update `Microsoft.WindowsTerminal` from Windows Termnal?

### DISCUSSION THREAD

**Discussion by sanamhub** [microsoft/winget-cli#1803] [General] [answered]
What is happening in my Windows Terminal is, other packages are getting updated except terminal. What are the workaround or solution for such type of issues?

Possible related issues
- Updating PowerShell from PowerShell
- Updating Terminal from Terminal and so on..

Screenshot

![image](https://user-images.githubusercontent.com/31341013/146133106-428e7b0c-4eb5-4705-9376-d113baaae447.png)

**Comment by Masamune3210** [comment]
Try updating terminal by itself, I know the update has to be postponed until you close the terminal but you shouldn't be getting a error like that, it should tell you something to the effect of letting you know that the update will be done once all Terminal windows are closed

**Comment by sanamhub** [comment, marked as answer]
Update is neither happening after closing terminal nor it's letting know that the update will be done once all terminal windows are closed :)
I've made a separate issue for this.. Thanks for the answer anyway!
